contrib/src/main/scala/akka/stream/contrib/Valve.scala

```diff
@@ -98,9 +105,18 @@ final class Valve[A](mode: SwitchMode) extends GraphStageWithMaterializedValue[F
           promise.success(succeed)
       }
 
+      // FIXME will never complete promise if stage is stopped, use invokeWithFeedback when Akka 2.5.7 is released
```

Comment:
Maybe it'd make sense to also already create a github issue to keep track of this... but this is probably sufficient for now :)

Reply:
will make sure to follow up on next release